pom.xml

```diff
@@ -144,6 +144,17 @@
             <artifactId>maven-surefire-plugin</artifactId>
             <version>2.22.2</version>
           </plugin>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-jar-plugin</artifactId>
+            <configuration>
+              <archive>
+                <manifestEntries>
+                  <Automatic-Module-Name>spotify.web.api.java</Automatic-Module-Name>
```

Comment:
Why this name?

Reply:
I used the artefact ID from pom.xml file.

Reply:
I read that the package name should be used here, which is currently `com.wrapper.spotify`. What are your thoughts on this? I haven't looked at Java 9 modules yet. What do you need them for?

Reply:
When I this without the module-info.java file, the code works perfectly, but when I add this file, I need to put the module name on this. The module name is defined on the module-info.java file, but to be compatible with older Java versions, the module name of a JAR is obtained from the manifest, and when absent, from the JAR name. I added this to be easier to include it on my Gradle project.

Reply:
So what about naming it `com.wrapper.spotify`? Do you think this would be a better idea?

Reply:
I think so.